Objects/longobject.c

```diff
@@ -6773,3 +6774,93 @@ int PyLong_AsUInt64(PyObject *obj, uint64_t *value)
 {
     LONG_TO_UINT(obj, value, "C uint64_t");
 }
+
+
+const PyLongLayout PyLong_LAYOUT = {
+    .bits_per_digit = PyLong_SHIFT,
+    .digits_order = -1,  // least significant first
+    .endian = PY_LITTLE_ENDIAN ? -1 : 1,
+    .digit_size = sizeof(digit),
+};
```

Comment:
I'm pretty sure that if you make this `static const ...` then you shouldn't need to add this to ignored.tsv.

Reply:
It fails with:

`Objects/longobject.c               	-                                  	PyLong_LAYOUT                           	not supported (typespec (unknown))	static const PyLongLayout`